GymApp/GymAppApi/Program.cs

```diff
@@ -1,8 +1,12 @@
+using GYM.API.Data;
+using Microsoft.EntityFrameworkCore;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 
 builder.Services.AddControllers();
+builder.Services.AddDbContext<GymDbContext>(opt => opt.UseSqlServer("name=ConnectionStrings:GymDbDefault"));
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
```

Comment:
Maybe it is better to use GetConnectionString() from IConfiguration interface. Smth like:

builder.Configuration.GetConnectionString("YourConnString")